admin-dev/themes/new-theme/public/theme.css

```diff
@@ -12747,3 +12747,8 @@ span.twitter-typeahead {
   width: 100% !important; }
   span.twitter-typeahead input.tt-input {
     width: 100% !important; }
+
+.stock-app table td {
```

Comment:
Hi @aleeks 
I know ;) 
But when i generate the assets, the "theme.css" file is modified.

Regards